docs/cli-arguments.md

```diff
@@ -54,6 +54,7 @@ Usage of ./kube-state-metrics:
       --telemetry-port int               Port to expose kube-state-metrics self metrics on. (default 8081)
       --tls-config string                Path to the TLS configuration file
       --total-shards int                 The total number of shards. Sharding is disabled when total shards is set to 1. (default 1)
+      --use-api-server-cache             Set resourceVersion=0 for ListWatch
```

Comment:
```suggestion
      --use-apiserver-cache             Sets resourceVersion=0 for ListWatch requests, using cached resources from the apiserver instead of an etcd quorum read.
```
rename the flag to be consistent with how we use apiserver in flags